component/text/text.css

```diff
@@ -0,0 +1,102 @@
+/* General */
+
+a {
+    text-decoration: none;
+}
+
+/* Typography */
+
+:root {
+    --font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue",
+    Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji",
+    "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
+/* Text Utilities */
+
+.heading1 {
+    font-size: 48px;
+    line-height: 64px;
```

Comment:
Can convert values > 10px to rem in all places.

Reply:
done